templates/aws/aws_cloudformation_template_oracle_rds.yml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
The name of the file needs to change.
`oracle_rds.cfn.template.yml`

Reply:
noted